SU2_CFD/src/python_wrapper_structure.cpp

```diff
@@ -31,865 +31,644 @@
 #include "../../Common/include/toolboxes/geometry_toolbox.hpp"
 
 void CDriver::PythonInterface_Preprocessing(CConfig **config, CGeometry ****geometry, CSolver *****solver){
-
-  int rank = MASTER_NODE;
-  SU2_MPI::Comm_rank(SU2_MPI::GetComm(), &rank);
-
-  /* --- Initialize boundary conditions customization, this is achieve through the Python wrapper --- */
-  for(iZone=0; iZone < nZone; iZone++){
-
-    if (config[iZone]->GetnMarker_PyCustom() > 0){
-
-      if (rank == MASTER_NODE) cout << endl << "----------------- Python Interface Preprocessing ( Zone "<< iZone <<" ) -----------------" << endl;
-
-      if (rank == MASTER_NODE) cout << "Setting customized boundary conditions for zone " << iZone << endl;
-      for (iMesh = 0; iMesh <= config[iZone]->GetnMGLevels(); iMesh++) {
-        geometry[iZone][INST_0][iMesh]->SetCustomBoundary(config[iZone]);
-      }
-      geometry[iZone][INST_0][MESH_0]->UpdateCustomBoundaryConditions(geometry[iZone][INST_0], config[iZone]);
-
-      if ((config[iZone]->GetKind_Solver() == MAIN_SOLVER::EULER) ||
-          (config[iZone]->GetKind_Solver() == MAIN_SOLVER::NAVIER_STOKES) ||
-          (config[iZone]->GetKind_Solver() == MAIN_SOLVER::RANS)) {
-
-        solver[iZone][INST_0][MESH_0][FLOW_SOL]->UpdateCustomBoundaryConditions(geometry[iZone][INST_0], config[iZone]);
-      }
+    
+    int rank = MASTER_NODE;
+    SU2_MPI::Comm_rank(SU2_MPI::GetComm(), &rank);
+    
+    /* --- Initialize boundary conditions customization, this is achieve through the Python wrapper --- */
+    for(iZone=0; iZone < nZone; iZone++){
+        
+        if (config[iZone]->GetnMarker_PyCustom() > 0){
+            
+            if (rank == MASTER_NODE) cout << endl << "----------------- Python Interface Preprocessing ( Zone "<< iZone <<" ) -----------------" << endl;
+            
+            if (rank == MASTER_NODE) cout << "Setting customized boundary conditions for zone " << iZone << endl;
+            for (iMesh = 0; iMesh <= config[iZone]->GetnMGLevels(); iMesh++) {
+                geometry[iZone][INST_0][iMesh]->SetCustomBoundary(config[iZone]);
+            }
+            geometry[iZone][INST_0][MESH_0]->UpdateCustomBoundaryConditions(geometry[iZone][INST_0], config[iZone]);
+            
+            if ((config[iZone]->GetKind_Solver() == MAIN_SOLVER::EULER) ||
```

Comment:
Is there a reason we didn't include the INC solver beforehand?   Should we add INC and NEMO here as well?

Reply:
I am not sure why they were not included before... but I've now added INC and NEMO support.